servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/H1ProtocolConfigBuilder.java

```diff
@@ -67,6 +67,9 @@ public H1ProtocolConfigBuilder headersFactory(final HttpHeadersFactory headersFa
      * @return {@code this}
      */
     public H1ProtocolConfigBuilder maxPipelinedRequests(final int maxPipelinedRequests) {
+        if (maxPipelinedRequests <= 0) {
+            throw new IllegalArgumentException("maxPipelinedRequests: " + maxPipelinedRequests + " (expected >0)");
```

Comment:
I will send a follow-up later to extract this validation into a common utility for ST

Reply:
👍 good idea

Reply:
My promised follow-up: https://github.com/apple/servicetalk/pull/2797